src/index.js

```diff
@@ -52,4 +68,17 @@ function checkStorage() {
   }
   return taskStorage;
 }
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  add(checkStorage(), document.getElementById('task').value);
+  form.reset();
```

Comment:
- [ ] The functionality to remove a todo item doesn't work as clicking on the delete icon has no effect, please fix it.

Reply:
the mentioned code shows add button but you asked to fix the remove button. I am a little bit confused. @mikenath223